be/src/exprs/time_functions.h

```diff
@@ -103,6 +103,21 @@ class TimeFunctions {
      */
     DEFINE_VECTORIZED_FN(day_of_week);
 
+    /**
```

Comment:
Could you give a syntax and example in the commit message?
It's better to describe how to use it in the commit message.

Reply:
ok，i would add these message and test